src/Makefile.gtest.include

```diff
@@ -44,7 +48,8 @@ zcash_gtest_SOURCES += \
 	gtest/test_libzcash_utils.cpp \
 	gtest/test_pedersen_hash.cpp \
 	gtest/test_checkblock.cpp \
-	gtest/test_zip32.cpp
+	gtest/test_zip32.cpp \
+	gtest/test_timedata.cpp
```

Comment:
It would be nice to insert this into roughly alphabetical order (I've been trying to sort it slowly as we add new tests).

Reply:
Why don't we just sort it fully, as a separate commit?

Reply:
Sure, that works.